src/storage.ts

```diff
@@ -236,6 +230,10 @@ export class Storage {
     return path.join(this.globalStorageUri.fsPath, "url")
   }
 
+  public getBinaryETag(): string {
+    return crypto.createHash("sha1").update(this.binaryPath()).digest("hex")
+  }
```

Comment:
I believe this is the same `sha1` hashing the backend is using. Although I'd love input on easiest way to check (less familiar on the Go side).

Reply:
SHA1 is universal, so as long as it's a SHA1 hash with hex encoding (lowercase) then it's good.